src/UIHandler.h

```diff
@@ -22,6 +22,7 @@ namespace Utils
 /**
  * Draws a border.
  *
+ * <pre>
```

Comment:
Yes, for doxygen.

Without <pre> it wraps my image into three weird lines. If you add <pre> (**pre**formated) it doesn't wrap it.